cognee/tests/test_pgvector.py

```diff
@@ -0,0 +1,72 @@
+
+
+import os
+import logging
+import pathlib
+import cognee
+from cognee.api.v1.search import SearchType
+
+logging.basicConfig(level=logging.DEBUG)
+
+async def main():
+    #TODO: Should this be set on pipeline side or in the test?
+    cognee.config.set_vector_engine_provider("pgvector")
```

Comment:
Let's refactor to have 1 method. Like `cognee.config.set_vector_db_config({...})` and `set_relational_db_config({...})`

Reply:
Done